src/__test__/breadcrumbstest.spec.tsx

```diff
@@ -1,13 +1,14 @@
-const { Builder, By} = require('selenium-webdriver')
-const {browser, sleep, n, options} = require("../setupTests");
-
 describe('Default Suite', function() {
   let driverCrumbs
+  var webdriver = require("selenium-webdriver");
```

Comment:
Have a look at our feature test configuration for Casematch: https://gitlab.eonum.ch/eonum/casematch/-/blob/master/spec/rails_helper.rb

Maybe there are other more suitable webdrivers and/or options.

Reply:
We use chromium in Casematch instead of Firefox.